packages/buffetjs-core/src/components/DatePicker/index.js

```diff
@@ -21,18 +21,18 @@ class DatePicker extends React.PureComponent {
   timer = null;
 
   componentDidMount() {
-    const { value, withDefaultValue } = this.props;
+    const { value, withDefaultValue, displayFormat } = this.props;
```

Comment:
I am not so sure about this one as it might be harder for the user to change the date when displaying it as `MMMM DD, YY` that's why I have hardcoded the format in the input. Maybe it can be handled using props for the input when it is opened? What are you thoughts?

Reply:
IMO it's unconsistenct. Especially because this is component library, which may be use elsewhere. My another purposal will be to add two formats.

And then with strapi it should be set by some kind of global settings or language specific (in translations).

Plus strapi should set moment.locale to admin panel language. Maybe I will add this with another PR ;)

Reply:
>  My another purposal will be to add two formats.

Sure that's what I had in mind